tests/appsec/test_asm_standalone.py

```diff
@@ -276,11 +350,19 @@ def test_any_upstream_propagation__with_attack__raises_priority_to_2__from_minus
                 ["Datadog-Client-Computed-Stats", trueish,] in data["request"]["headers"] for trueish in ["yes", "true"]
             )
 
+        downstream_headers = json.loads(self.r.text)
+        assert downstream_headers["X-Datadog-Origin"] == "rum"
+        assert downstream_headers["X-Datadog-Parent-Id"] != "34343434"
+        assert "_dd.p.other=1" not in downstream_headers["X-Datadog-Tags"]
```

Comment:
I don't understand this assertion about the absence of _dd.p.other=1 while it wasn't present in the weblog request. Isn't it always necessarily true? (the comment applies to the different places where this is done)

Reply:
Yes, it's redundant